### DISCUSSION TITLE
Exit Code 1 - Moodle website does not work PLAYER yuja.com

### DISCUSSION THREAD

**Discussion by djunite1** [aclap-dev/video-downloadhelper#393] [Help and Q&A]
### Your question

Hey

I'm using Moodle website for my University and it does not work

Their Player is "YUJA" - which this player used to work in the past!

Exit code: 1
Error: Exit code: 1

    at as.receive (chrome-extension://lmjnegcaeklhafolokijcfjliaokphfk/background/main.js:1:13157)
    at chrome-extension://lmjnegcaeklhafolokijcfjliaokphfk/background/main.js:1:42624
    
    Can someone tell me why it stopped working and how to fix it !?

### Addon & CoApp version and details

Video DownloadHelper
Version 8.2.2.9 (stable)
Browser locale: en-US
Production build
Built on 2024-03-19
Build options: browser=chrome, linuxlic=false, noyt=true, target=google

**Comment by djunite1** [comment]
HIT details:

let me know what else you need!

<header class="weh-header" style="box-sizing: border-box; display: block; min-height: 42px; line-height: 42px; background-repeat: no-repeat; background-position: 6px 6px; background-size: 30px; flex: 0 0 auto; background-color: rgb(248, 248, 248); color: rgb(33, 37, 41); font-family: -apple-system, &quot;system-ui&quot;, &quot;Segoe UI&quot;, Roboto, &quot;Helvetica Neue&quot;, Arial, &quot;Noto Sans&quot;, sans-serif, &quot;Apple Color Emoji&quot;, &quot;Segoe UI Emoji&quot;, &quot;Segoe UI Symbol&quot;, &quot;Noto Color Emoji&quot;; font-size: 16px; font-style: normal; font-variant-ligatures: normal; font-variant-caps: normal; font-weight: 400; letter-spacing: normal; orphans: 2; text-align: left; text-indent: 0px; text-transform: none; widows: 2; word-spacing: 0px; -webkit-text-stroke-width: 0px; white-space: normal; text-decoration-thickness: initial; text-decoration-style: initial; text-decoration-color: initial; background-image: url(&quot;images/icon-128.png&quot;);"><span class="weh-header-close" style="box-sizing: border-box; font-weight: 100; font-size: 1.5em; line-height: 1.5em; cursor: pointer; color: gray; padding: 0px 12px 0px 0px; float: right;"><br class="Apple-interchange-newline">⊗</span></header><main style="box-sizing: border-box; display: block; flex: 1 1 auto; position: relative; overflow-y: auto; height: 622px; color: rgb(33, 37, 41); font-family: -apple-system, &quot;system-ui&quot;, &quot;Segoe UI&quot;, Roboto, &quot;Helvetica Neue&quot;, Arial, &quot;Noto Sans&quot;, sans-serif, &quot;Apple Color Emoji&quot;, &quot;Segoe UI Emoji&quot;, &quot;Segoe UI Symbol&quot;, &quot;Noto Color Emoji&quot;; font-size: 16px; font-style: normal; font-variant-ligatures: normal; font-variant-caps: normal; font-weight: 400; letter-spacing: normal; orphans: 2; text-align: left; text-indent: 0px; text-transform: none; widows: 2; word-spacing: 0px; -webkit-text-stroke-width: 0px; white-space: normal; background-color: rgb(255, 255, 255); text-decoration-thickness: initial; text-decoration-style: initial; text-decoration-color: initial;">
bitrate | 600000
-- | --
chunked | hls
core_media | {...}
created | 1712801385453
descrPrefix | HLS streaming
duration | 2679.48
extension | mp4
group | grp-76dfd2e3
headers | [...]
id | hls:317e6263
isPrivate | false
masterManifest | https://d3k4p2smfceopy.cloudfront.net/P/Data/VideoUrl/Video1462bf81389d4bfc8dc9e6927e939bed_processedmp4_-index.m3u8?dist=cz1-chatter&key=Video-1462bf81-389d-4bfc-8dc9-e6927e939bed_processed.mp4_hls/index.m3u8&classPID=83803&cookie=true&expiration=1712812182&signature=03218546a85e86bbf2292aef6534c6b27c566c4251b562b07e0628e29f527211
mediaManifest | https://d3k4p2smfceopy.cloudfront.net/Video-1462bf81-389d-4bfc-8dc9-e6927e939bed_processed.mp4_hls/360p/Video-1462bf81-389d-4bfc-8dc9-e6927e939bed_processed.mp4_hls.m3u8
originalId | network-probe:4cae8534
pageUrl | https://concordia.yuja.com/V/Video?isPlaying=false&startTime=285&node=5436515&z=1&u=561c7264-558a-42f9-a2b5-dbc22602aabc
referrer | https://concordia.yuja.com/V/Video?isPlaying=false&startTime=285&node=5436515&z=1&u=561c7264-558a-42f9-a2b5-dbc22602aabc
status | inactive
tabId | 1760140366
thumbnailUrl | https://concordia.yuja.com/P/DataPage/BroadcastsThumb/vvxnP0b1YW
title | Zoom #3 2024.03.14
topUrl | https://concordia.yuja.com/V/Video?isPlaying=false&startTime=285&node=5436515&z=1&u=561c7264-558a-42f9-a2b5-dbc22602aabc

</main>

**Comment by djunite1** [comment]
I also tried the beta version!

I used to use Video DownloadHelper for over 2+ years without any issues!

**Comment by Mattmjs** [comment]
It does encounter the error in the extension like you say. I don't know why it happens since it works directly in ffmpeg.
I got 5 minutes out of the 44 minute video before quitting the download. It might also work in yt-dlp.

**Reply by djunite1** [reply]
Can you make a video and/or explain how to fix it; as if not I have found another app that does the trick; so weird that this app has been working for 2 years and all of the sudden just stops working!

Was great; I hope to see a fix soon or I'll be forced to switch over!

**Reply by djunite1** [reply]
No idea what is ffmpeg nor ty-dlp!

**Reply by Mattmjs** [reply]
ffmpeg is what the companion app comes packaged with, when you click something to download in the extension ffmpeg
performs the download. While it encounters the error in the extension, using ffmpeg directly it handles it no problem.
yt-dlp is another separate tool not related to VDH. If you've found something that's works then just use that.
I'm just another user.
